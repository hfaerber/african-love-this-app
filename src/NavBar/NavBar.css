.nav-header {
  width:98%;
  text-align: center;
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.div-nav-btns {
  display: flex;
}
