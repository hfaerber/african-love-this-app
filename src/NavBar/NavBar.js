import React from 'react';
import './NavBar.css';
import { NavLink } from 'react-router-dom';
import { NavBtn } from '../NavBtn/NavBtn';

export const NavBar = () => {
  return(
    <header className='nav-header'>

      <h2 className="App-header">African
      <span role='img' aria-label='heart emoji' className='header-heart'>♥️</span>
      This App</h2>
      <div className='div-nav-btns'>
        <NavBtn text={'Home'} link={'/'}/>
        <NavBtn text={'Study'} link={'/study'}/>
        <NavBtn text={'Quiz'} link={'/quiz'}/>
      </div>
    </header>
  )
}

export default NavBar;
