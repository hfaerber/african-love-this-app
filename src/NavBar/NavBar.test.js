import React from 'react';
import { NavBar } from './NavBar';
import { shallow } from 'enzyme';

describe('NavBar', () => {
  let wrapper;

  it('should match snapshot', () => {
    wrapper = shallow(<NavBar
      text='Quiz'
      link='./quiz'
    />);
    expect(wrapper).toMatchSnapshot();
  })
})
