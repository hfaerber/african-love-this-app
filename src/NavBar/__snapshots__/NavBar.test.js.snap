// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match snapshot 1`] = `
<header
  className="nav-header"
>
  <h2
    className="App-header"
  >
    African
    <span
      aria-label="heart emoji"
      className="header-heart"
      role="img"
    >
      ♥️
    </span>
    This App
  </h2>
  <div
    className="div-nav-btns"
  >
    <NavBtn
      link="/"
      text="Home"
    />
    <NavBtn
      link="/study"
      text="Study"
    />
    <NavBtn
      link="/quiz"
      text="Quiz"
    />
  </div>
</header>
`;
